website/content/en/docs/deploy-guide.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
I think we probably want to frame this in terms of "Installation" rather than "Deployment". Deployment is a reserved term in the kubernetes world, and it may mislead users. This is pretty standard for other projects:

https://istio.io/latest/docs/setup/install/
https://argoproj.github.io/argo-workflows/installation/